torch/_refs/nn/functional/__init__.py

```diff
@@ -419,6 +420,32 @@ def hinge_embedding_loss(
     return _apply_loss_reduction(loss, reduction)
 
 
+# TODO: Cannot register as decomp here due to incompatible signatures between
+# the Python frontend and ATen:
+# https://github.com/pytorch/pytorch/issues/83931
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("input", "target"),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def huber_loss(
+    input: TensorLikeType,
+    target: TensorLikeType,
+    reduction: str = "mean",
```

Comment:
the only difference is in reduction argument? I think we can make reference accept both string and integer reduction argument.

Reply:
How about doing it the other way around, and have torch reference map a str argument to an int, and the call _decomp aten decomposition?

Reply:
> How about doing it the other way around, and have torch reference map a str argument to an int, and the call _decomp aten decomposition?

It doesn't really matter which way it would be done today